CodeStarterCamp_Week1/main.swift

```diff
@@ -1,31 +1,152 @@
 //
-//  main.swift
 //  CodeStarterCamp_Week1
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by 임경인 on 2022/04/21.
 //
 
 import Foundation
 
-let star: String = "*"
-let stick: String = "| |"
+// 첫번째 빼빼로
 
-func printIce(makeStar: String) {
-    for _ in 1...8 {
-            for _ in 1...11 {
-                print("\(makeStar)", terminator: "")
+let firstBody: String = "***"
+let firstStick: String = "| |"
+let firstStickHigh: Int = 4
+let bodyHigh: Int = 10
+let firstToping: String = ""
 
+//몸통과 토핑을 매개변수로 받아 몸통을 그리는 함수
+func printFirstBody(body: String, toping: String) {
+    print("\(toping)","\(body)")
+    }
+
+//길이에 따른 막대를 그리는 함수
+func printFirstStick(length: Int) {
+    for _ in 1...length {
+        print("","\(firstStick)")
+    }
+}
+
+//길이에 따라 몸통을 그리는 함수
+func printMainBody(high: Int) {
+    for _ in 1...high {
+        printFirstBody(body: firstBody, toping: firstToping)
+    }
+}
+
+func firstPpePpeRo() {
+    printMainBody(high: bodyHigh)
+        printFirstStick(length: firstStickHigh)
+}
+
+firstPpePpeRo()
+
+//----------------------------------------------------------------------------------------
+
+// 2번째 빼빼로
+
+let seceondBody: String = "***"
+let seceondStick: String = "| |"
+let seceondStickHigh: Int = 4
+let seceondBodyHigh: Int = 12
+let seceondToping: String = "&"
+
+//몸통과 토핑을 매개변수로 받아 몸통을 그리는 함수
+func seceondBody(body: String, topping: String) {
+    print("\(topping)","\(body)","\n",
+            " ","\(body)","\(topping)",separator: "")
+}
+//길이에 따른 막대를 그리는 함수
+func printSeceondStick(length: Int) {
+    for _ in 1...length {
+        print("","\(seceondStick)")
+    }
+}
+
+//길이에 따라 몸통을 그리는 함수
+func printSeceondMainBody(high: Int) {
+    for _ in 1...high/2 {
+        seceondBody(body:seceondBody,topping:seceondToping)
+    }
+}
+func seceondPpePpeRo() {
+    printSeceondMainBody(high: seceondBodyHigh)
+        printSeceondStick(length: seceondStickHigh)
+}
+
+seceondPpePpeRo()
+
+//--------------------------------------------------------------------------------
+//세번째 빼빼로
+let thirdBody: String = "***"
+let thirdStick: String = "| |"
+let thirdStickHigh: Int = 4
+let thirdBodyHigh: Int = 12
+let thirdToping: String = "#"
+
+//몸통과 토핑을 매개변수로 받아 몸통을 그리는 함수
+
+func printThirdBody(body: String, topping: String) {
+    print("\(topping)","\(body)","\n",
+            " ","\(body)","\(topping)",separator: "")
```

Comment:
보통 이렇게 코드가 길어서 가독성을 위해 내려쓰는 경우에는
이런식으로 많이 쓰는 것 같더라구요!
```suggestion
    print(
        "\(topping)\(body)\n \(body)\(topping)",
        separator: ""
    )
```

콤마는 안써도 될 것 같아서 지워보았어요. 저는 오히려 콤마때문에 가독성이 안좋은 것 같은 느낌이 드네요. 🤔

Reply:
ㅠㅠ 지금이라도 \()뒤에 콤마를 안써도 되는걸 알았습니다..너무감사합니다 주신 코드가 너무 깔끔하네요 전부다 수정을 해야겠어요!!!😊😊 감사해용~